### PR TITLE
Keymap for Budgy Keyboard

### PR DESCRIPTION
**Pull request by JakobEdvardsson**
This config has both US-qwerty and Swedish-Colemak-DH in mind.  
The default keymap is US-qwerty, in order to switch to the Swedish-Colemak-DH keymap,
comment out the US-qwerty keymap and uncomment the Swedish-Colemak-DH keymap in the main.py file.

### CODE REVIEW

File: boards/budgy/qwerty/kb.py
Comment (xs5871) [inline boards/budgy/qwerty/kb.py:1]: Why does qwerty need a seperate `kb.py`?

Comment (JakobEdvardsson) [reply boards/budgy/qwerty/kb.py:1]: Will fix, did not mean to do that

**Review by xs5871 — COMMENTED** [review]
If you're providing two keymaps, wouldn't it make more sense to have common `kb.py` and `main.py`, and split off only the keymaps as `keymap_sw.py` and `keymap_us.py` for example?

**Comment by JakobEdvardsson** [comment]
> If you're providing two keymaps, wouldn't it make more sense to have common `kb.py` and `main.py`, and split off only the keymaps as `keymap_sw.py` and `keymap_us.py` for example?

You are right! Will make sure to do that instead!

**Comment by JakobEdvardsson** [comment]
@xs5871 made the keymap more modular, all you have to do is to uncomment/comment out an import to change layout.

**Comment by xs5871** [comment]
There are linter errors that need fixing

**Review by xs5871 — APPROVED** [review]

**Pull request merged**